source/cli/appservices-function-run.txt

```diff
@@ -62,6 +62,22 @@ Options
      - string
      - false
      - Specify which user to run the function as (Note: Using <none> will run as the System user) (Default value: <none>; Allowed values: <none>, <userID>)
```

Comment:
~I'm confused why this and/or the `-y` option wasn't updated as part of [DOCSP-38465](https://jira.mongodb.org/browse/DOCSP-38465) - i.e. depending on how those are set and there are available users, the CLI prompts for the user to run the function as~
looking at the other CLI docs, it seems like we don't provide that level of detail?? I don't know if I agree with that standard...

Reply:
Yeah it seems they don't document all of the flags, as it might be overwhelming on the page (this command alone has about 6 flags)